packages/firebase_performance/analysis_options.yaml

```diff
@@ -1,11 +1,107 @@
-# This is a temporary file to allow us to land a new set of linter rules in a
-# series of manageable patches instead of one gigantic PR. It disables some of
-# the new lints that are already failing on this plugin, for this plugin. It
-# should be deleted and the failing lints addressed as soon as possible.
-
-include: ../../analysis_options.yaml
```

Comment:
silly question probably, but why (if the all_lint_rules.yaml and analysis_options.yaml are effectively boilerplate duplicated amongst packages) don't they just refer to a common file in firebase_core or similar?

Reply:
The goal is, after these PRs are merged, I'll make a separate PR removing all the duplicates that at the root of the repo

Reply:
makes sense as a pragmatic approach, I was sincerely just curious. Thanks @rrousselGit